### PR TITLE
Generate random customer group for sales_flat_order.customer_group_id.

### PR DESCRIPTION
**Pull request by scirner22**
### Motivation
Closes: https://github.com/RJMetrics/magenerator/issues/36

### Implementation Notes
- Alex tells me customer_group_id should be a random customer group
- Added Dockerfile, didn't want to install these libraries on my host

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    
+RUN npm install chance && \
+    npm install should && \
+    npm install random-date && \
+    npm install csv-parse && \
+    npm install async
+
+COPY ["generate.coffee", "/opt/app"]
+
+CMD ["coffee"]
```
Comment (b-ryan) [inline Dockerfile]: Oh snap, dawg

File: README.md [hunk 2]
```diff
@@ -5,6 +5,13 @@ Contact: Ben Garvey bgarvey@magento.com
 
 License: OSL 3.0
 
+# Docker Setup
+If you have docker installed
+```
+docker build -t magenerator .
+docker run -v `pwd`/data:/opt/app/data magenerator coffee generate.coffee
```
Comment (b-ryan) [inline README.md]: Booo this man. Never ` always $()

**Comment by b-ryan** [comment]
Seems reasonable to me. I don't know anything about this repo. If it works gopher it.

**Comment by bengarvey** [comment]
cool
:+1:

**Pull request merged**